components/card/index.jsx

```diff
@@ -0,0 +1,47 @@
+import { createElement } from "complate-stream";
+
+export function Card(_, ...children) {
+	return <div class="card">{ children }</div>;
+}
+
+export function CardImg({ src, alt }, ...children) {
+	return <img class="card-img-top" src={ src } alt={ alt }/>;
+}
+
+export function CardHeader({ level = "4" }, ...children) {
```

Comment:
Card Headers can be attached to any element, though I believe only block-level elements are sensible to use. Maybe `level` should not have a fallback value, and we would use `div` if it’s undefined?

Reply:
Yep, I think that makes sense. I would not allow arbitrary elements – `h*` and `div` should be enough.